src/test/java/com/nurkiewicz/java8/J07_StreamsTest.java

```diff
@@ -76,12 +76,15 @@ public void areAllPeopleNotSlim() {
 		assertThat(allNotSlim).isFalse();
 	}
 
+	/**
```

Comment:
No, `comparing()` is introduced in subsequent lesson `com.nurkiewicz.java8.J08_FilesTest`. Here old-style `Comparator` will probably be used (replaced with lambda) just to feel the pain :-).